src/libs/actions/Policy/Policy.ts

```diff
@@ -2036,15 +2036,45 @@ function openPolicyTaxesPage(policyID: string) {
     API.read(READ_COMMANDS.OPEN_POLICY_TAXES_PAGE, params);
 }
 
-function openPolicyExpensifyCardsPage(policyID: string) {
+function openPolicyExpensifyCardsPage(policyID: string, workspaceAccountID: number) {
     const authToken = NetworkStore.getAuthToken();
 
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.SHARED_NVP_PRIVATE_EXPENSIFY_CARD_SETTINGS}${workspaceAccountID}`,
```

Comment:
For all of these, we can stop using the SHARED_NVP
```suggestion
            key: `${ONYXKEYS.COLLECTION.PRIVATE_EXPENSIFY_CARD_SETTINGS}${workspaceAccountID}`,
```

Reply:
This update is handled in another [PR](https://github.com/Expensify/App/pull/46564/files#diff-facfc70b32eba19ccaeeadd16b51faf6dbf2ceb919373565e7322c86047886cd)